### PR TITLE
Frontend container migration

### PR DESCRIPTION
**Pull request by jrusz**
Adding necessary scripts for migrating our frontend component to container.

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="image-builder"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/image-builder-frontend"
```
Comment (ochosi) [inline pr_check.sh:9]: Out of curiosity, does this container have to be built manually with the build_deploy.sh script? Or will this end up in a GitHub Action?

Comment (jrusz) [reply pr_check.sh:9]: It should be handled by the @app-sre-bot you can see the results of it already in the checks here as `ci.ext.devshift.net PR build` :)

Comment (ezr-ondrej) [reply pr_check.sh:9]: yeah, it's automatically built by jenkins :) `build_deploy.sh` on merge to main and `pr_check.sh` on every PR push.

**Comment by jrusz** [comment]
I think this is good now, worked fine when I tried deploying it. @croissanne could you take a look at this please?

**Review by croissanne — APPROVED** [review]
lgtm, thank you!

**Review by croissanne — APPROVED** [review]

**Pull request merged**